extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -504,24 +504,34 @@ export class AmpConsent extends AMP.BaseElement {
       ) {
         this.updateCacheIfNotNull_(
           response['consentStateValue'],
-          response['consentString'] || undefined
+          response['consentString'] || undefined,
+          response['gdprApplies'] !== undefined
+            ? response['gdprApplies']
+            : !!response['consentRequired']
```

Comment:
@zhouyx , We've talked about this before, but I think another thing to consider is that there is no way to tell if the `gdprApplies` signal was sent from CMP or vendor or if it was defaulted to `consentRequired`. 

I.e. the case when user is in California, but `gdprApplies` is not provided in `checkConsentHref`, so then `gdprApplies` will be true, even though use is in Cali. 

Do you think this is a valid concern?

Reply:
It's really a tradeoff between more tagging effort to specify the `gdprApplies`, or more tagging effort to not specify the `gdprApplies`.
I'm leaning towards making `gdprApplies` to match `consentRequired` to minimize changes to the server, so the variable would work out of the box. 
I understand your concern to not make the assumption. But since we've received agreement on the proposal from CMP and ad vendors, I'd say that's fine.

Reply:
Thanks for the explanation. SGTM